<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeScreen</class>
 <widget class="QWidget" name="WelcomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: palette(dark);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0" colspan="6">
      <widget class="QLabel" name="label">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt; color: palette(base);</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="6">
      <widget class="QListWidget" name="lstRecentProjects">
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	text-decoration: underline;
	color: palette(midlight);
	background-color: palette(mid);
	selection-background-color: palette(mid);
}

QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="batchSize">
        <number>10</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0" rowspan="2" colspan="4">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt; color: palette(base);</string>
       </property>
       <property name="text">
        <string>Recent Projects</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="5" column="4">
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="5" column="5">
      <widget class="QPushButton" name="btClearRecentProjects">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>32</height>
        </size>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Clear the recent projects list</string>
       </property>
       <property name="statusTip">
        <string>Clear the recent projects list</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	text-decoration: underline;
	color: palette(base);
	border: none;
}

QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="5">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="5">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1" colspan="2">
      <widget class="QPushButton" name="btImportData">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Import a spectra list and compute scores</string>
       </property>
       <property name="statusTip">
        <string>Import a spectra list and compute scores</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	text-decoration: underline;
	color: palette(midlight);
	border: none;
}

QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>Import Data</string>
       </property>
       <property name="icon">
        <iconset resource="../ui.qrc">
         <normaloff>:/icons/images/import-mgf.svg</normaloff>:/icons/images/import-mgf.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QPushButton" name="btOpenProject">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="toolTip">
        <string>Load a previously saved project file</string>
       </property>
       <property name="statusTip">
        <string>Load a previously saved project file</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
	text-decoration: underline;
	color: palette(midlight);
	border: none;
}

QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>Open Project</string>
       </property>
       <property name="icon">
        <iconset resource="../ui.qrc">
         <normaloff>:/icons/images/document-open.svg</normaloff>:/icons/images/document-open.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="3" colspan="3">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_16">
       <property name="maximumSize">
        <size>
         <width>10</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="9" column="2">
      <widget class="QLabel" name="label_17">
       <property name="statusTip">
        <string>https://dx.doi.org/10.1021/acs.analchem.9b02802</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://dx.doi.org/10.1021/acs.analchem.9b02802&quot; style=&quot;text-decoration: underline; color: palette(midlight);&quot;&gt;Anal. Chem. 2019, 91, 18, 11489-11492&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_12">
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ui.qrc">:/icons/images/git.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_11">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ui.qrc">:/icons/images/browser.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ui.qrc">:/icons/images/help.svg</pixmap>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <spacer name="verticalSpacer_5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <widget class="QLabel" name="label_7">
       <property name="statusTip">
        <string>https://github.com/metgem/metgem</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://github.com/metgem/metgem&quot; style=&quot;text-decoration: underline; color: palette(midlight);&quot;&gt;Source Code&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_14">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ui.qrc">:/icons/images/academic.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ui.qrc">:/icons/images/academic.svg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_5">
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string>https://metgem.readthedocs.io</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://metgem.readthedocs.io&quot; style=&quot;text-decoration: underline; color: palette(midlight);&quot;&gt;User Manual&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="10" column="1" colspan="2">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_6">
       <property name="statusTip">
        <string>https://metgem.readthedocs.io/getting_started.html</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://metgem.readthedocs.io/getting_started.html&quot; style=&quot;text-decoration: underline; color: palette(midlight);&quot;&gt;Getting Started&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_8">
       <property name="statusTip">
        <string>https://metgem.github.io</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://metgem.github.io&quot; style=&quot;text-decoration: underline; color: palette(midlight);&quot;&gt;MetGem Website&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../ui.qrc">:/icons/images/help.svg</pixmap>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLabel" name="label_15">
       <property name="statusTip">
        <string>https://dx.doi.org/10.1021/acs.analchem.8b03099</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>&lt;a href=&quot;https://dx.doi.org/10.1021/acs.analchem.8b03099&quot;style=&quot;text-decoration: underline; color: palette(midlight);&quot;&gt;Anal. Chem. 2018, 90, 23, 13900-13908&lt;/a&gt;</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0" colspan="3">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt; color: palette(base);</string>
       </property>
       <property name="text">
        <string>Community</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="3">
      <widget class="QLabel" name="label_18">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt; color: palette(base);</string>
       </property>
       <property name="text">
        <string>Publications</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="7" column="0" colspan="3">
      <widget class="QLabel" name="label_13">
       <property name="styleSheet">
        <string notr="true">color: palette(midlight);</string>
       </property>
       <property name="text">
        <string>If you use MetGem for your research, please cite</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1" rowspan="2">
      <widget class="QCheckBox" name="chkEnableNews">
       <property name="toolTip">
        <string>Show news about MetGem: this needs internet to retrieve information from MetGem's website</string>
       </property>
       <property name="statusTip">
        <string>Show news about MetGem: this needs internet to retrieve information from MetGem's website</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QCheckBox
{
	color: palette(base);
}

QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="text">
        <string>Enabled</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" rowspan="2">
      <widget class="QLabel" name="lblNews">
       <property name="styleSheet">
        <string notr="true">font-size: 20pt; color: palette(base);</string>
       </property>
       <property name="text">
        <string>News</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QListWidget" name="lstNews">
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">QListWidget
{
	color: palette(midlight);
	background-color: palette(mid);
	selection-background-color: palette(mid);
}

QToolTip
{
	color: palette(tooltiptext);
	background-color: palette(tooltip);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Adjust</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ui.qrc"/>
 </resources>
 <connections/>
</ui>
